docs/pypi-replication.md

```diff
@@ -56,6 +56,37 @@ map                   	all	0.2805
 recall_1000           	all	0.9470
 ```
 
+MS MARCO passage ranking task, dense retrieval with TCT-ColBERT, HNSW index.
```

Comment:
Can you move this to a new file, like `docs/dense-retrieval.md`? This feature is still experimental... I don't want to mix it with pretty stable stuff, like the rest of the docs here...

Reply:
sure